packages/web-twig/src/Resources/components/RadioField/stories/RadioField.twig

```diff
@@ -0,0 +1,11 @@
+<RadioField
+    id="radiofield0"
+    isChecked={true}
```

Comment:
As in Checkbox, this is probably useless to set it true.
```suggestion
    isChecked
```

Reply:
Done